mr/src/main/java/org/apache/iceberg/mr/Catalogs.java

```diff
@@ -83,28 +90,33 @@ private static Table loadTable(Configuration conf, String tableIdentifier, Strin
               .impl(catalogLoaderClass)
               .build()
               .newInstance();
-      return Optional.of(loader.load(conf));
+      Catalog catalog = loader.load(conf);
+      LOG.info("Dynamic catalog is used: [{}]", catalog);
```

Comment:
I think it would be better to give more specific information:

```java
LOG.info("Loaded catalog {} using {}", catalog, catalogLoaderClass);
```

Reply:
Done